debian/control

```diff
@@ -23,7 +23,6 @@ Build-Depends:
  libsoup2.4-dev,
  libsystemd-dev,
  ostree,
- xsltproc,
```

Comment:
This commit can be squashed into the main one. No need to commit to the `debian/` directory separately.